src/solutions/two-fer/two-fer.js

```diff
@@ -0,0 +1,33 @@
+import { always, curry, ifElse, isNil } from 'ramda';
+
+/* Version 1 */
+
+// twoFer :: String -> String
+// const twoFer = name => {
```

Comment:
Are default parameters really more simple? I definitely don't think this is black and white, but I wonder  whether it is important to make a distinction between when and when not to have a fallback value for a function param because a default param means there are more ways that a function can be called:

```javascript
const twoFer = (name = 'you') => `One for ${name}, one for me.`;

console.log(twoFer('Muhammad')); // One for Muhammad, one for me. ✅ 
console.log(twoFer()); // One for you, one for me. ✅ 
console.log(twoFer(undefined)); One for you, one for me. ✅ 
console.log(twoFer('')); // One for , one for me. ❌  
console.log(twoFer(null)); // One for null, one for me. ❌ 
```